tests/cpp-tests/Classes/PhysicsTest/PhysicsComponentTest.cpp

```diff
@@ -201,10 +208,12 @@ Sprite* PhysicsComponentDemo::makeTriangle(Vec2 point, Size size, int color, Phy
         triangle->setScaleY(size.height/43.5f);
     }
     
-    Vec2 vers[] = { Vec2(0, size.height/2), Vec2(size.width/2, -size.height/2), Vec2(-size.width/2, -size.height/2)};
-    
-    auto body = PhysicsBody::createPolygon(vers, 3, material);
-    addPhysicsComponent(triangle, body);
+    //Vec2 vers[] = { Vec2(0, size.height/2), Vec2(size.width/2, -size.height/2), Vec2(-size.width/2, -size.height/2)};
+	Vec2 vers[] = { Vec2(0, triangle->getContentSize().height / 2), Vec2(triangle->getContentSize().width / 2, -triangle->getContentSize().height / 2), Vec2(-triangle->getContentSize().width / 2, -triangle->getContentSize().height / 2) };
```

Comment:
the values `size.heght` and `size.width` you used in `vers[]` can not correctly express the body's size